### PR TITLE
und-625

### PR DESCRIPTION
**Pull request by Ashish-srij**
filter and search

### CODE REVIEW

File: stories/Patterns/FilterAndSearchBar/FilterAndSearchBar.scss [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
   border: 0;
   border-bottom: 1px solid $color-black;
   cursor: pointer;
   padding: 0 0 0.625rem;
   text-decoration: underline;
-
-  .icon-search-black {
-    height: 1.75rem;
-  }
+  background: #fff url(#{$img-path-Iconback}/Search_black.svg) no-repeat 0 0/1.75rem;
```
Comment (surbhi90) [inline stories/Patterns/FilterAndSearchBar/FilterAndSearchBar.scss:12]: @Ashish-srij please use color code

File: stories/Patterns/FilterAndSearchBar/FilterAndSearchBar.scss [hunk 2]
```diff
@@ -1,19 +1,17 @@
-@import '../../assets/scss/mixins';
-@import '../../assets/scss/breakpoints';
-@import '../../assets/scss/variables';
+@import "../../assets/scss/mixins";
+@import "../../assets/scss/breakpoints";
+@import "../../assets/scss/variables";
```
Comment (surbhi90) [inline stories/Patterns/FilterAndSearchBar/FilterAndSearchBar.scss:3]: @Ashish-srij Please use the proper linting as it should have single quotes

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Patterns/FilterAndSearchBar/FilterAndSearchBar.scss (hunk 1 above)
Comment (Ashish-srij) [reply stories/Patterns/FilterAndSearchBar/FilterAndSearchBar.scss]: @surbhi90 Done.

File: stories/Patterns/FilterAndSearchBar/FilterAndSearchBar.scss (hunk 2 above)
Comment (Ashish-srij) [reply stories/Patterns/FilterAndSearchBar/FilterAndSearchBar.scss]: @surbhi90 Done.

**Pull request merged**